components/src/core/components/SidePanel/Branding.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <a class="oxd-brand" :class="classes" :href="url">
+    <div class="oxd-brand-icon">
+      <img alt="orangehrm logo" src="../../../assets/images/orange.png" />
```

Comment:
when we move to orangehrm side, this will not work and can't change image URL. it's better to expose prop to setup the `src` externally

Reply:
added src prop. updated stories.